features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/services/jagg.jag

```diff
@@ -0,0 +1,28 @@
+<%
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+    include("/services/constants.jag");
+
+    var utils = Packages.org.wso2.carbon.apimgt.impl.utils.APIUtil;
+    var serverPort = utils.getManagementTransportPort(MGT_TRANSPORT.split(":")[0]);
```

Comment:
does this work when there is a url without a port? say LB address

Reply:
The method `.getManagementTransportPort()` returns the port for the given transport type, value of `MGT_TRANSPORT` is set to "https://" in constants.jag.